defaults/main.yml

```diff
@@ -79,6 +79,9 @@ tomcat_configure_webapps: "{{ tomcat_configure }}"
 tomcat_extra_libs_path: []
 tomcat_webapps_path: []
 
+#Removes ROOT webapp Context if false
+tomcat_root_context: true
```

Comment:
Could we make this variable a bit more self explanatory. 

`tomcat_remove_root_context` or `tomcat_remove_root_context_on_install`